romanisim/bandpass.py

```diff
@@ -181,3 +182,26 @@ def compute_count_rate(flux, bandpass, filename=None, effarea=None, wavedist=Non
     # per second
 
     return zpflux
+
+
+def etomjysr(bandpass):
```

Comment:
Yeah, I think I lost track of the fact that this depends on pixel scale and that leads to issues when doing L3 simulations with different pixel scales.  Conceptually I think I want this to be for the native pixel scale---what number of electrons per pixel corresponds to a MJy/sr?  But I need to think more about how to do this.

Reply:
I've now changed the definition of the exposure time to be the total time on the field rather than the time adjusted by the pixfrac ** 2.  This was (is?) confusing because GalSim internally accounts for the different pixel scale when rendering sources, but for things like the sky the handling is conceptually a bit different.  The most important changes I ended up making along these lines were:
* when calibrating back to MJy/sr, include the factor of the pixfrac^2; this now is in etomjysr when passed to simulate_cps.  Maybe it would have been better to include it as an argument here, though!
* The thermal backgrounds are in pixel units rather than area units, and need to get a factor of pixfrac^2
* The read noise is also a per-pixel quantity and needs to be adjusted when pixfrac != 1.  It gets larger since the effective time on each output pixel is smaller.
* I added some scaling tests intended to exercise this flavor of thing.